### PR TITLE
Added code necessary to dockerize the application. The docker instanc…

### PR DESCRIPTION
**Pull request by evindj**
…e will not run the DB instance.

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -6,4 +6,5 @@
 app = create_app("development")
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=8000, debug=True)
+    test = os.environ['TEST']
```
Comment (evindj) [inline run.py]: remove this test

File: README.md [hunk 2]
```diff
@@ -14,9 +14,8 @@ This is a small API written in python
 - To drop DB, run `manage dropDB`
 
 ## To run
-0. Run Script to create DATABASES  `.\db/create.sh`
-1. Create virtual environment and activate it and then `pip install -r requirements.txt`.
-2. To launch the server run the command: `python run.py`
+0. Create virtual environment and activate it and then `pip install -r requirements.txt`.
```
Comment (mkenlo) [inline README.md]: I guess there is not more need for a virtual environment, since we are dockerizing the app.

File: README.md [hunk 3]
```diff
@@ -14,9 +14,8 @@ This is a small API written in python
...
+1. To launch the server run the command: `python run.py`
```
Comment (mkenlo) [inline README.md]: How do we launch now the app? What are the steps to launch the container hence the app?

File: .gitignore [hunk 4]
```diff
@@ -4,3 +4,5 @@ env
 __pycache__
 app/__pycache__
 .pytest_cache
+.dockerfile.swp
```
Comment (mkenlo) [inline .gitignore:7]: Are files dockerfile with small 'd' and dockerfile Capital 'D' different?

**Review by mkenlo — APPROVED** [review]

**Pull request merged**